ResourceStatusSystem/Command/CEAvailabilityCommand.py

```diff
@@ -0,0 +1,80 @@
+# $HeadURL:  $
+''' CEAvailabilityCommand module 
+'''
+
+from DIRAC                                                      import S_OK, S_ERROR
+from DIRAC.ResourceStatusSystem.Client.ResourceManagementClient import ResourceManagementClient
+from DIRAC.ResourceStatusSystem.Command.Command                 import Command
+from DIRAC.Interfaces.API.DiracAdmin                            import DiracAdmin
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry          import getVOs
+
+__RCSID__ = '$Id:  $'
+
+class CEAvailabilityCommand( Command ):
+  '''
+    It returns the status of a given CE.
+  ''' 
+
+  def __init__( self, args = None ):
+    '''
+      
+      :params: args: dictionary specifying following values
+        :attr:`ce`: Computing Element (mandatory)  
+        :attr:`host`: BDII server (optional)
+    '''
+    super( CEAvailabilityCommand, self ).__init__( args )
+  
+  def doCommand( self ):
+    '''  
+      It returns the status of a given CE.
+      
+      :return:
+        a dictionary with status of each CE queues, 
+        and 'status' and 'reason' of the CE itself     
+       
+    '''
+    
+    ## INPUT PARAMETERS
+    vo = getVOs()['Value'].pop()
```

Comment:
Since it uses getSection, it can return S_ERROR, so you should check the return type